packages/cli/src/ui/commands/chatCommand.ts

```diff
@@ -142,7 +142,7 @@ const saveCommand: SlashCommand = {
     }
 
     const history = chat.getHistory();
-    if (history.length > 0) {
+    if (history.length > 2) {
```

Comment:
as a follow up, it would it be more robust if chat had a method that determined whether the history had any messages that were actually from the user? It is slightly paranoid but I do worry this constant might change from 2 to some other value in the future if we tweaked how the initial conversation is setup.

Reply:
@jacob314 Thanks for the suggestion.

I agree that relying on a hardcoded constant for the minimum history length is fragile, especially if the initial conversation setup changes in the future. I think moving this logic into some function that checks for actual user messages would make the code more robust and maintainable.

It also makes sense to consider refactoring the resume command to use similar logic for consistency. I will take this up and propose a separate refactor PR, making future changes easier to manage.